.tina/collections/logos.tsx

```diff
@@ -30,6 +30,11 @@ export const logosSchema: Collection = {
       },
       templates: [...Schemas.pageBlocks],
     },
+    {
+      type: "boolean",
+      name: "showBorderBottom",
+      label: "Show Border Bottom",
+    },
```

Comment:
I think you might be able to make `<hr>`s in markdown, might mean you don't need this if you use the footer `rich-text`

Reply:
yes, rich text does allow that - i have updated it to use the rich text instead of a separate boolean field + html